src/Form/EmbedButtonForm.php

```diff
@@ -61,7 +60,16 @@ public function form(array $form, FormStateInterface $form_state) {
     $form = parent::form($form, $form_state);
 
     $embed_button = $this->entity;
-    $button_image = $embed_button->button_icon_fid ? array($embed_button->button_icon_fid) : array();
+
+    // Get default for button image. If its uuid is set, get the id of the file
+    // to be used as default in the form.
+    $button_icon = array();
+    if ($embed_button->button_icon_uuid) {
+      $files = $this->entityManager->getStorage('file')->loadByProperties(array('uuid' => $embed_button->button_icon_uuid));
```

Comment:
Or at least $this->entityManager()->loadEntityByUuid('file', $this->button_icon_uuid) like above in calculateDependencies()